DDDetectors/src/Mask_o1_v01_geo.cpp

```diff
@@ -58,6 +69,9 @@ static dd4hep::Ref_t create_element(dd4hep::Detector& description,
   if (xmlParameter.hasAttr(_Unicode(rotationX)))
     rotationX = xmlParameter.attr< bool >(_Unicode(rotationX));
 
+  int counter = 0; 
+  Material sectionMat;
+  bool isSensitive = false;
   for(xml_coll_t c( xmlMask ,Unicode("section")); c; ++c) {
```

Comment:
I think this should work:
```suggestion
  for(xml_coll_t c( xmlMask ,Unicode("section")); c; ++c, ++counter) {
```
And then drop the increments from inside the loop body